package.json

```diff
@@ -45,6 +45,7 @@
     "vinyl-source-stream": "^1.0.0",
     "vinyl-transform": "^1.0.0",
     "webmaker-auth": "1.0.2",
-    "webmaker-login-ux": "git://github.com/mozilla/webmaker-login-ux#master"
+    "webmaker-login-ux": "git://github.com/mozilla/webmaker-login-ux#master",
+    "webmaker-analytics": "~0.1.7"
```

Comment:
minor issue: can we move this to above webmaker-auth so things stay alphabetically ordered?